### ISSUE TITLE
Add support for inline comments

### ISSUE THREAD

**Issue by cwbit** [jacobbednarz/atom-language-ini#26]
Hi! Any way to get this to support inline comments?

```ini
; this works fine

files[] = blah.txt     ;but this comment does not
```

**Comment by jacobbednarz** [comment]
This has been raised before in #10. I'm still not sure whether I'd like it in the package seeing how loose the INI spec is and I'm not sure whether this would cause more confusion for those languages that don't support it.

**Comment by jacobbednarz** [comment]
Alright, I think I'm going to have to take a different approach here because INI is so loose. Perhaps a better approach here would be to try and cover all the implementation variants (such as commenting) and only revert to the base implementation in the event of a parser conflict.

**Comment by jacobbednarz** [comment]
Released under v1.16.0 :sparkles:
